examples/catalog/src/content/Content.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {TopAppBarFixedAdjust} from '@material/react-top-app-bar';
+
+import {MenuContext} from '../Context';
+import {Menus} from '../Router';
+
+export const Content = () => (
+  <MenuContext.Consumer>
+    {({selectedIndex}) => (
+      <TopAppBarFixedAdjust>
+        {React.createElement(Menus[selectedIndex].component)}
```

Comment:
This seems like an antipattern. Sorry I didn't pick this up before. What is going on here. Is this the recommended usage?

Reply:
I don't know this is a recommendation.
But I know that JSX is compiled like this.
```
<MyComponent></MyComponent> => React.createElement(MyComponent, someProps)
```
And this is the doc. (https://reactjs.org/docs/react-without-jsx.html)

Reply:
Ok fair enough! :)